src/Products/Common/Entity/Web/LoadDocumentEntity.cs

```diff
@@ -12,25 +12,25 @@ public class LoadDocumentEntity
         /// Document absolute path.
         /// </summary>
         [JsonProperty]
-        private string guid;
+        public string guid;
```

Comment:
Codacy found an issue: [Make 'guid' private.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5958475)